docs/initrd.md

```diff
@@ -0,0 +1,21 @@
+# Building a custom initrd and using it in a mkosi image
+
+To build an image with a mkosi-built initrd:
+1. Build an initrd image. You can for example build a **compressed** `cpio`
+archive with mkosi for any distribution containing the packages `systemd` and `udev`:
```

Comment:
Maybe a bit more assertive
```suggestion
1. Build an initrd image using the `cpio` output format with the same target distribtution as you want to use for your distribution image. mkosi compresses the `cpio` output format by default.
```
I'm not sure the notes on udev and systemd are necessary, since we already pull them in everywhere, no?

Reply:
We don't pull in udev by default unless it's a bootable image

Reply:
Ah, brain fart on my side, I thought systemd would generally pull in udev, but of course, it doesn't. 

But then maybe we should just pull udev in automatically when building initrds, because I guess we'll only want to support initrds with udev and systemd as init?